src/views/partials/__footer.njk

```diff
@@ -5,28 +5,22 @@
+            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/contact" data-track-options="{&quot;dimension29&quot;:&quot;Contact&quot;}" href="https://www.gov.uk/government/organisations/companies-house#org-contacts">Contact Us</a>
           </li>
           <li class="govuk-footer__inline-list-item">
-            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/help/accessibility-statement" data-track-options="{&quot;dimension29&quot;:&quot;Accessibility statement&quot;}" href="/help/accessibility-statement">Accessibility statement</a>
+            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/accessibility" data-track-options="{&quot;dimension29&quot;:&quot;Accessibility statement&quot;}" href="/accessibility-statement">Accessibility statement</a>
           </li>
           <li class="govuk-footer__inline-list-item">
-            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/contact" data-track-options="{&quot;dimension29&quot;:&quot;Contact&quot;}" href="/contact">Contact</a>
+            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/developers" data-track-options="{&quot;dimension29&quot;:&quot;Developers&quot;}" href="/developers">Developers</a>
           </li>
           <li class="govuk-footer__inline-list-item">
-            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/help/terms-conditions" data-track-options="{&quot;dimension29&quot;:&quot;Terms and conditions&quot;}" href="/help/terms-conditions">Terms and conditions</a>
-          </li>
-          <li class="govuk-footer__inline-list-item">
-            <a class="govuk-footer__link" lang="cy" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/cymraeg" data-track-options="{&quot;dimension29&quot;:&quot;Rhestr o Wasanaethau Cymraeg&quot;}" href="/cymraeg">Rhestr o Wasanaethau Cymraeg</a>
-          </li>
-          <li class="govuk-footer__inline-list-item">
-            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/government/organisations/government-digital-service" data-track-options="{&quot;dimension29&quot;:&quot;Government Digital Service&quot;}" href="/government/organisations/government-digital-service">Government Digital Service</a>
+            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/clear" data-track-options="{&quot;dimension29&quot;:&quot;Clear&quot;}" href="/manage-prototype/clear-data">Clear data</a>
```

Comment:
Same with this, the clear button is prototype only, the prototype is referenced in the link